apps/sensenet/src/components/view-controls/browse-view.tsx

```diff
@@ -55,6 +58,7 @@ export const BrowseView: React.FC<BrowseViewProps> = (props) => {
         handleCancel={() => navigateToAction({ history, routeMatch })}
         controlMapper={controlMapper}
         localization={{ cancel: localization.forms.cancel }}
+        locale={personalSettings.language === 'hungarian' ? hu : enUS}
```

Comment:
we should create a function on LocalizationContext, which returns the locale (this approach is not sustainable if we have more languages available).

Reply:
I used an object there to define the languages, the langcodes and the locale. I hope it is also good :)